src/modules/function.js

```diff
@@ -0,0 +1,73 @@
+import { addTask, updateLocalStorage } from './app.js';
```

Comment:
It is recommended to avoid modifying arrays directly, as it can cause unexpected behavior. Consider creating a new array with the remaining tasks instead of modifying the original array in the removeTask() and clearCompletedTasks() functions.